packages/next-app/src/hooks/useTransaction.ts

```diff
@@ -100,6 +107,11 @@ export const NOTIFICATION_MAP_INTERNAL_BALANCES = {
     description: "Your transaction is being made",
     variant: NotificationVariant.NOTIFICATION,
   },
+  [TransactionStatus.CONFIRMING]: {
+    title: "Great!",
+    description: "The approval was a success!",
```

Comment:
```suggestion
    description: "The approval was successful!",
```
the copies sound a bit off to me -- I can help out with that later on but would take a read at how other apps are doing it.

Reply:
Thanks! =)